fe/src/components/base/tabs.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="flex overflow-y-auto space-x-4">
+    <div
+      v-for="tab in items"
+      :key="tab"
+      :class="`rounded-md h-11 ${
+        tab === activeTab
+          ? 'bg-gradient-to-r from-[#CF7A1E] to-[#AC3971] p-[1px] cursor-not-allowed'
```

Comment:
Notice that this style gradient style is repeating. 
This is where we can utilize the tailwind.css custom class. 
Transfer this gradient style to tailwind.css file and create a custom class for it.